src/js/createKeyboard/index.js

```diff
@@ -0,0 +1,25 @@
+import {helperMessage} from "../globalVariables";
+
+export default class CreateKeyboard {
```

Comment:
Bad naming - 'Create' word is verb - it is good for functions, but not for classes.
Possible names: `class Keyboard` or `class KeyboardCreator'